velox/common/memory/ByteStream.cpp

```diff
@@ -204,6 +204,17 @@ void ByteStream::seekp(std::streampos position) {
   VELOX_FAIL("Seeking past end of ByteStream: {}", position);
 }
 
+size_t ByteStream::flushSize() {
+  updateEnd();
```

Comment:
Is this a copy-paste from 'flush'? Let's refactor to avoid copy-paste.

Reply:
It is different from `flush` because `flush` will call `write` in the for loop, it is hard to refactor.